.circleci/config.yml

```diff
@@ -1,7 +1,6 @@
 version: 2.1
 orbs:
   matlab: mathworks/matlab@0
-  codecov: codecov/codecov@1
```

Comment:
Were we showing code coverage at some point and then ended up taking it out?

Reply:
I think this was a copy-past remnant. I don't believe we were ever showing code coverage with codecov in these examples.